.eslintrc

```diff
@@ -28,7 +28,10 @@
 				"varsIgnorePattern": "^detectWeb$|^doWeb$|^detectImport$|^doImport$|^doExport$|^detectSearch$|^doSearch$"
 			}
 		],
-		
+
+		"no-redeclare": ["error", {"builtinGlobals": true}],
+		"no-global-assign": "error",
```

Comment:
`no-global-assign` is already included by way of `eslint:recommended` in `@zotero/eslint-config` (as is `no-redeclare`, but without `"builtinGlobals": true`), so is can be removed here.